docs/about/all/support/guidelines-for-gdpr-compliance.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Guidelines for GDPR compliance
```

Comment:
Who's the target audience of this doc?

Reply:
Answered in - https://spryker.slack.com/archives/C3KMWM1UZ/p1713176881720259?thread_ts=1712141501.987429&cid=C3KMWM1UZ